### DISCUSSION TITLE
signal-cli for raspberrypi zero w

### DISCUSSION THREAD

**Discussion by NOYB4Europe** [AsamK/signal-cli#685] [Q&A]
Please do not really see that as an "issue" than a polite question. 
I would like to run signal-cli on a raspberrypi zero w. Now java-11 does not run on a raspberrypi zero w and signal-cli does require java-11. Is there any way to get signal-cli to run on java-8 ... or get a stand alone build for the "zero w"? Am I maybe missing something? 
Thank you ! 

Just after writing here I had the idea to revert to 0.6.8 (last version before JRE11 became required) I am testing that now.

**Comment by AsamK** [comment]
signal-cli uses lots of java 11 features, so running it on java 8 is not possible.

**Comment by technillogue** [comment]
you could try using the native-image
